ml/automl/tasks/__init__.py

```
@@ -0,0 +1,67 @@
+    'download_dataset_from_minio',
+    'train_model',
+    'validate_trained_model',
+    'upload_model_to_minio',
+    # Model deployment tasks
+    'get_production_model_info',
+    'update_production_pointer',
+    'update_latest_checkpoint',
+    'emit_deployment_event',
+    # Artifact tasks
+    'create_pipeline_artifact'
+]
```

Ruff (0.14.3)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)